src/SDKs/HDInsight/Management/Management.HDInsight/Management.HDInsight.csproj

```diff
@@ -7,11 +7,11 @@
 		<PackageId>Microsoft.Azure.Management.HDInsight</PackageId>
 		<Description>Azure HDInsight Management SDK Library</Description>
 		<AssemblyName>Microsoft.Azure.Management.HDInsight</AssemblyName>
-		<Version>4.2.0-preview</Version>
+		<Version>4.3.0-preview</Version>
```

Comment:
We need to take a look at the versioning story here, this is inconsistent with our versioning guidelines

Reply:
Thanks for helping review this. Change the version number to 4.2.1-preview.